### ISSUE TITLE
v1.0.2 doesn't seem to re-attempt npm install on ECONNRESET

### ISSUE THREAD

**Issue by Puneeth-n** [jfromaniello/npm-install-retry#10]
```
00:00:51.772 attempt 1: npm install --loglevel=error
00:00:56.229 npm ERR! Linux 3.19.0-66-generic
00:00:56.229 npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--loglevel=error"
00:00:56.229 npm ERR! node v4.3.2
00:00:56.229 npm ERR! npm  v2.14.12
00:00:56.229 npm ERR! code ECONNRESET
00:00:56.229 npm ERR! errno ECONNRESET
00:00:56.229 npm ERR! syscall read
00:00:56.229 
00:00:56.229 npm ERR! network read ECONNRESET
00:00:56.230 npm ERR! network This is most likely not a problem with npm itself
00:00:56.230 npm ERR! network and is related to network connectivity.
00:00:56.230 npm ERR! network In most cases you are behind a proxy or have bad network settings.
00:00:56.230 npm ERR! network 
00:00:56.230 npm ERR! network If you are behind a proxy, please make sure that the
00:00:56.230 npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
00:00:57.375 should-sinon@0.0.5 node_modules/should-sinon
00:00:57.375 
00:00:57.375 async@1.5.2 node_modules/async
00:00:57.375 
00:00:57.375 http-aws-es@1.1.3 node_modules/http-aws-es
00:00:57.375 
00:00:57.375 serverless-helpers-js@0.0.3 node_modules/serverless-helpers-js
00:00:57.375 └── dotenv@1.2.0
00:00:57.375 
00:00:57.375 coffee-script@1.10.0 node_modules/coffee-script
00:00:57.375 
00:00:57.375 should@8.4.0 node_modules/should
00:00:57.375 ├── should-type@0.2.0
00:00:57.375 ├── should-equal@0.8.0
00:00:57.375 └── should-format@0.3.2
00:00:57.375 
00:00:57.375 sinon@1.17.5 node_modules/sinon
00:00:57.375 ├── formatio@1.1.1
00:00:57.375 ├── samsam@1.1.2
00:00:57.375 ├── lolex@1.3.2
00:00:57.375 └── util@0.10.3 (inherits@2.0.1)
00:00:57.375 
00:00:57.375 mocha@2.5.3 node_modules/mocha
00:00:57.375 ├── escape-string-regexp@1.0.2
00:00:57.375 ├── commander@2.3.0
00:00:57.375 ├── diff@1.4.0
00:00:57.375 ├── supports-color@1.2.0
00:00:57.375 ├── growl@1.9.2
00:00:57.375 ├── to-iso-string@0.0.2
00:00:57.375 ├── debug@2.2.0 (ms@0.7.1)
00:00:57.375 ├── mkdirp@0.5.1 (minimist@0.0.8)
00:00:57.375 ├── glob@3.2.11 (inherits@2.0.3, minimatch@0.3.0)
00:00:57.375 └── jade@0.26.3 (commander@0.6.1, mkdirp@0.3.0)
00:00:57.375 
00:00:57.375 jsonwebtoken@5.7.0 node_modules/jsonwebtoken
00:00:57.375 ├── ms@0.7.1
00:00:57.375 ├── xtend@4.0.1
00:00:57.375 └── jws@3.1.3 (jwa@1.1.3, base64url@1.0.6)
00:00:57.375 
00:00:57.375 aws-sdk@2.2.33 node_modules/aws-sdk
00:00:57.375 ├── xmlbuilder@0.4.2
00:00:57.375 ├── xml2js@0.2.8
00:00:57.375 └── sax@0.5.3
00:00:57.375 
00:00:57.375 elasticsearch@10.1.3 node_modules/elasticsearch
00:00:57.375 ├── forever-agent@0.6.1
00:00:57.375 ├── promise-js@0.0.6
00:00:57.375 ├── chalk@1.1.3 (escape-string-regexp@1.0.5, supports-color@2.0.0, ansi-styles@2.2.1, strip-ansi@3.0.1, has-ansi@2.0.0)
00:00:57.375 ├── lodash@3.10.1
00:00:57.375 └── lodash-compat@3.10.2
00:00:57.375 
00:00:57.375 request@2.74.0 node_modules/request
00:00:57.375 ├── aws-sign2@0.6.0
00:00:57.375 ├── tunnel-agent@0.4.3
00:00:57.375 ├── forever-agent@0.6.1
00:00:57.375 ├── oauth-sign@0.8.2
00:00:57.375 ├── is-typedarray@1.0.0
00:00:57.375 ├── caseless@0.11.0
00:00:57.375 ├── stringstream@0.0.5
00:00:57.375 ├── aws4@1.4.1
00:00:57.375 ├── isstream@0.1.2
00:00:57.375 ├── json-stringify-safe@5.0.1
00:00:57.375 ├── extend@3.0.0
00:00:57.375 ├── tough-cookie@2.3.1
00:00:57.375 ├── qs@6.2.1
00:00:57.375 ├── node-uuid@1.4.7
00:00:57.375 ├── combined-stream@1.0.5 (delayed-stream@1.0.0)
00:00:57.375 ├── mime-types@2.1.11 (mime-db@1.23.0)
00:00:57.375 ├── hawk@3.1.3 (cryptiles@2.0.5, boom@2.10.1, sntp@1.0.9, hoek@2.16.3)
00:00:57.375 ├── bl@1.1.2 (readable-stream@2.0.6)
00:00:57.375 ├── http-signature@1.1.1 (assert-plus@0.2.0, jsprim@1.3.1, sshpk@1.10.0)
00:00:57.375 ├── har-validator@2.0.6 (pinkie-promise@2.0.1, commander@2.9.0, chalk@1.1.3, is-my-json-valid@2.13.1)
00:00:57.375 └── form-data@1.0.1 (async@2.0.1)
00:00:57.375 npm ERR! Linux 3.19.0-66-generic
00:00:57.375 npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--loglevel=error"
00:00:57.375 npm ERR! node v4.3.2
00:00:57.375 npm ERR! npm  v2.14.12
00:00:57.375 
00:00:57.375 npm ERR! Callback called more than once.
00:00:57.375 npm ERR! 
00:00:57.375 npm ERR! If you need help, you may report this error at:
00:00:57.375 npm ERR!     <https://github.com/npm/npm/issues>
00:00:57.375 
00:00:57.375 npm ERR! Please include the following file with any support request:
00:00:57.376 npm ERR!     /home/somedir/npm-debug.log
00:00:57.390 Makefile:4: recipe for target 'init' failed
```

**Comment by Puneeth-n** [comment]
I see PR #8 fixed this issue.

**Comment by jfromaniello** [comment]
Sorry about that, released as `v1.1.0`.
